src/packages/bem/bem.js

```diff
@@ -99,20 +99,25 @@ function bemStateless(blockDecl) {
                 classnamesMap: classnames,
             });
         },
-        elem: (elemName, props, extraMods = {}) => {
+        elem: (elemName, props, classesToKeep = '') => {
             if (!elemName || !props) {
                 throw new TypeError(
                     'elem(elemName, props) should be called with elem name and props as an arguments'
                 );
             }
-            return buildBemProps({
-                block: name,
-                elem: elemName,
-                props,
-                propsToMods,
-                extraMods,
-                classnamesMap: classnames,
-            });
+            const classes = [
+                buildBemProps({
+                    block: name,
+                    elem: elemName,
+                    props,
+                    propsToMods,
+                    extraMods: {},
```

Comment:
Now it doesn't make sense to leave `extraMods` in ` buildBemProps`, should be removed from there as well.

Reply:
removed from everywhere